src/runs/executions.ts

```diff
@@ -104,7 +111,7 @@ export const deleteBinary = (language: Language, binPath: string) => {
         if (platform() == 'linux') {
             spawn('rm', [binPath]);
         } else {
-            spawn('del', [binPath]);
+            spawn('del', [binPath], { shell: true });
```

Comment:
Currently, the binary/class files are not getting deleted on Windows. This seems to work. I'll look into this a little more to pinpoint the exact issue. Shall I open this as a separate issue?

Reply:
Oh - since I dont use Windows I wasn't aware of this issue. Please do open another issue.

Reply:
Yeah, I have created #87 for this and linked the same to this PR.